charts/matomo/values.yaml

```diff
@@ -0,0 +1,37 @@
+mariadb:
+  image: image-registry.apps.2.rahti.csc.fi/openshift/mariadb:10.3-el8
+  name: mariadb
+  service:
+    type: ClusterIP
+  pvc:
+    storageSize: 5Gi
+    storageClassName: standard-csi
+  secret:
+    databaseName: matomodb
+    databaseUser: matomouser
+    databasePassword: letmein
+    rootPassword: letmein
```

Comment:
Could these password be random?

Reply:
Fixed. Password are defined randomly in the `_helpers.tpl` file